plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -780,9 +791,9 @@ entryExitTracing lamName displayName e ty =
         annMayInline
         ( mkTrace
             (PLC.TyFun annMayInline defaultUnitTy ty) -- ()-> ty
-            ("entering " <> displayName)
+            ("-> " <> displayName)
```

Comment:
I see that this special strings are used in several places:
`plutus-core/executables/traceToStacks/Common.hs`
and here.

I suggest to define them once (as a binding) and then refer to their defined names from all places using them. This helps static code navigation, makes the link more explicit.

Reply:
@effectfully what is your opinion on this matter?